packages/itwin/tree-widget/src/components/trees/models-tree/internal/ModelsTreeVisibilityHandler.ts

```diff
@@ -465,77 +520,17 @@ class ModelsTreeVisibilityHandlerImpl implements HierarchyVisibilityHandler {
       return EMPTY;
     }
 
-    return this.changeElementState({
-      elementId: node.key.instanceKeys[0].id,
+    return this.changeElementsState({
+      elementIds: new Set([node.key.instanceKeys[0].id]),
```

Comment:
Let's try to be more correct, since we're now capable of handling multiple IDs at once.


```suggestion
      elementIds: new Set([...node.key.instanceKeys.map(({ id }) => id)]),
```